### DISCUSSION TITLE
it is 2021 not 2008 Please Improve this website like https://redux-toolkit.js.org/

### DISCUSSION THREAD

**Discussion by amantiwari1** [encode/django-rest-framework#7935] [Ideas & Suggestions]
i hate every old website and poor UI/UX, i don't understand they don't focus documentation on this website to caused unpopular package
for example, `` package is so popular due to they have a better new website,  I could improve this website or anyone can do it because we are the community we must help improve their website, agree?

**Comment by lovelydinosaur** [comment]
Yeah updating the docs is kinda on the radar right now.

Tho ya know, maybe try to be a bit more *constructive* in how you raise issues? 💛
